### PR TITLE
align destroyers code base for better readability

### PR DESCRIPTION
**Pull request by mjudeikis**
tide-up destroyers code

* move all aws discovery into the aws sub-package to match all other destroyers 
* Move `New` function to the top to be closer to main struct. Improves code readability.

### CODE REVIEW

**Comment by abhinavdahiya** [comment]
Please follow guidelines for commits https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format

**Comment by abhinavdahiya** [comment]
> tide up destroyers code

What do you mean by tide up? What standards are you trying to follow, is it a refactor for better readability or usability?

> This aligns all destroyers code to be the same `pkg` structure as all other destroyers.

It looks like you are moving the aws destroyer discovery to aws pkg, its that's the goal please be specific.

**Comment by mjudeikis** [comment]
added descriptions

**Comment by abhinavdahiya** [comment]
> added descriptions

> Please follow guidelines for commits https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format

I still do not see the commit messages updated to match the guidelines.

File: pkg/destroy/libvirt/libvirt.go [hunk 1]
```diff
@@ -68,6 +68,15 @@ func (o *ClusterUninstaller) Run() error {
 	return nil
 }
 
+// New returns libvirt Uninstaller from ClusterMetadata.
```
Comment (abhinavdahiya) [inline pkg/destroy/libvirt/libvirt.go]: All other New funcs are the first function after the uninstraller func in this PR, maybe follow that for this too?

Comment (mjudeikis) [reply pkg/destroy/libvirt/libvirt.go]: resolved

**Pull request merged**